src/ethereumjs-wallet.d.ts

```diff
@@ -0,0 +1,18 @@
+declare module 'ethereumjs-wallet' {
```

Comment:
If this covers the whole module, do you plan on adding it to their repo at some point?

Reply:
Not sure if it's full coverage but good idea